modules/apps/foundation/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/search/bar/portlet/action/SearchBarRedirectMVCActionCommand.java

```diff
@@ -77,6 +82,8 @@ protected void doProcessAction(
 			ActionRequest actionRequest, ActionResponse actionResponse)
 		throws Exception {
 
+		List selectedFacetSites = new ArrayList();
```

Comment:
While testing this solution to populate site facets in the redirect URL, ran into an issue where the last selected facet would populate upon search even after unselecting facet. May need to figure out how to clear the variable _selectedSites if needed.